platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterUtils.java

```diff
@@ -36,4 +63,145 @@ private RosterUtils() {}
     public static String formatNodeName(final long nodeId) {
         return "node" + (nodeId + 1);
     }
+
+    /**
+     * Hashes the given {@link Roster} object.
+     *
+     * @param roster the roster to hash
+     * @return the hash of the roster
+     */
+    @NonNull
+    public static Hash hashOf(@NonNull final Roster roster) {
```

Comment:
This hash method now has to go as @anthony-swirldslabs is working on pushing a pbj version of hasher right now.